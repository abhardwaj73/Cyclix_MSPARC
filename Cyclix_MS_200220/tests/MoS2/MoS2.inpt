CELL: 29.910212616 0.369599135716446 6.055850878784
FD_GRID: 150 53 31
KPOINT_GRID: 1 17 1
CHEB_DEGREE: 50
BC: D C P
EXCHANGE_CORRELATION: GGA_PBE
TOL_SCF: 1e-5
TOL_POISSON: 1e-7
TOL_PSEUDOCHARGE: 1e-7
SMEARING: 0.001
ELEC_TEMP_TYPE: fd
CALC_STRESS: 0
SPIN_TYP: 1
