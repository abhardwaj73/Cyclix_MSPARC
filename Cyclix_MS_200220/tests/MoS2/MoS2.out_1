***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 05-May-2020  Start time: 14:02:12                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.910213 0.369599 6.055851 
FD_GRID: 150 53 31
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 17 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 50
NSTATES: 36
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/MoS2/MoS2.out_1
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.199401 (Bohr)
Mesh spacing in y-direction        :  0.195493 (Bohr)
Mesh spacing in z direction        :  0.195350 (Bohr)
Number of symmetry adapted k-points:  9
Output printed to                  :  ../tests/MoS2/MoS2.out_1
Total number of atom types         :  2
Total number of atoms              :  6
Total number of electrons          :  52
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ./Mo.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.50 0.21 6.64
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ./S.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.50 0.21 6.64
Number of atoms of type 2          :  4
Estimated total memory usage       :  5.47 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9644890331E+01        1.014E-01        403.935
2            -2.9673404540E+01        1.120E-01        115.155
3            -2.9686449974E+01        2.275E-02        110.181
4            -2.9687187005E+01        2.098E-02        108.498
5            -2.9687561203E+01        1.137E-02        104.832
6            -2.9687689267E+01        3.847E-03        106.825
7            -2.9687687833E+01        2.916E-03        103.489
8            -2.9687709729E+01        1.592E-03        107.419
9            -2.9687713565E+01        5.961E-04        104.132
10           -2.9687714668E+01        2.167E-04        102.093
11           -2.9687715151E+01        1.751E-04        98.913
12           -2.9687715707E+01        8.567E-05        101.597
13           -2.9687715669E+01        3.238E-05        104.866
14           -2.9687714889E+01        2.257E-05        104.273
15           -2.9687715491E+01        1.118E-05        98.588
16           -2.9687715161E+01        7.597E-06        99.973
Total number of SCF: 16    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9687715161E+01 (Ha/atom)
Total free energy                  : -1.7812629096E+02 (Ha)
Band structure energy              : -4.2549122925E+01 (Ha)
Exchange correlation energy        : -2.4734395411E+01 (Ha)
Self and correction energy         : -3.5605006960E+02 (Ha)
Entropy*kb*T                       : -2.0221758751E-07 (Ha)
Fermi level                        : -1.6558745589E-01 (Ha)
Average force                      :  3.6425106864E-02 (Ha/Bohr)
Maximum force                      :  8.0963881979E-02 (Ha/Bohr)
Time for force calculation         :  30.316 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  2046.036 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
