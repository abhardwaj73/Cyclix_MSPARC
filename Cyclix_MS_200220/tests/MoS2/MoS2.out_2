***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 05-May-2020  Start time: 15:49:15                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.910213 0.369599 6.055851 
FD_GRID: 150 53 31
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 17 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 50
NSTATES: 36
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/MoS2/MoS2.out_2
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.199401 (Bohr)
Mesh spacing in y-direction        :  0.195493 (Bohr)
Mesh spacing in z direction        :  0.195350 (Bohr)
Number of symmetry adapted k-points:  9
Output printed to                  :  ../tests/MoS2/MoS2.out_2
Total number of atom types         :  2
Total number of atoms              :  6
Total number of electrons          :  52
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ./Mo.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.50 0.21 6.64
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ./S.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.50 0.21 6.64
Number of atoms of type 2          :  4
Estimated total memory usage       :  5.47 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9645103641E+01        1.004E-01        408.528
2            -2.9673686395E+01        1.100E-01        109.911
3            -2.9686231398E+01        2.333E-02        107.734
4            -2.9686999520E+01        2.099E-02        109.523
5            -2.9687355407E+01        1.168E-02        103.813
6            -2.9687479676E+01        3.888E-03        109.316
7            -2.9687478086E+01        2.823E-03        105.037
8            -2.9687495278E+01        1.623E-03        107.538
9            -2.9687497169E+01        6.349E-04        105.070
10           -2.9687495675E+01        2.387E-04        106.419
11           -2.9687495206E+01        1.897E-04        102.055
12           -2.9687494608E+01        1.009E-04        105.223
13           -2.9687494303E+01        3.659E-05        103.461
14           -2.9687493324E+01        2.412E-05        101.837
15           -2.9687493924E+01        1.487E-05        102.629
16           -2.9687493474E+01        5.984E-06        103.064
Total number of SCF: 16    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9687493474E+01 (Ha/atom)
Total free energy                  : -1.7812496084E+02 (Ha)
Band structure energy              : -4.2567220167E+01 (Ha)
Exchange correlation energy        : -2.4733193121E+01 (Ha)
Self and correction energy         : -3.5605006960E+02 (Ha)
Entropy*kb*T                       : -2.0228380132E-07 (Ha)
Fermi level                        : -1.6593893894E-01 (Ha)
Average force                      :  3.8685333637E-02 (Ha/Bohr)
Maximum force                      :  8.3873395280E-02 (Ha/Bohr)
Time for force calculation         :  31.114 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  2063.783 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
