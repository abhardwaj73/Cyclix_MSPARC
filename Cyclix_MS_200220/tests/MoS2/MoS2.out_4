***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 06-May-2020  Start time: 00:03:17                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.910213 0.369599 6.055851 
FD_GRID: 150 53 31
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 17 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 50
NSTATES: 36
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/MoS2/MoS2.out_4
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.199401 (Bohr)
Mesh spacing in y-direction        :  0.195493 (Bohr)
Mesh spacing in z direction        :  0.195350 (Bohr)
Number of symmetry adapted k-points:  9
Output printed to                  :  ../tests/MoS2/MoS2.out_4
Total number of atom types         :  2
Total number of atoms              :  6
Total number of electrons          :  52
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ./Mo.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.50 0.21 6.64
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ./S.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.50 0.21 6.64
Number of atoms of type 2          :  4
Estimated total memory usage       :  5.47 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9644805655E+01        1.020E-01        408.332
2            -2.9673188290E+01        1.138E-01        110.720
3            -2.9686449945E+01        2.248E-02        109.378
4            -2.9687185719E+01        2.099E-02        109.772
5            -2.9687563482E+01        1.134E-02        105.137
6            -2.9687691628E+01        3.870E-03        109.481
7            -2.9687690164E+01        2.908E-03        105.125
8            -2.9687712405E+01        1.573E-03        109.013
9            -2.9687716265E+01        5.891E-04        106.275
10           -2.9687717435E+01        2.169E-04        105.214
11           -2.9687717947E+01        1.750E-04        101.685
12           -2.9687718485E+01        8.579E-05        104.399
13           -2.9687718445E+01        3.219E-05        102.136
14           -2.9687717687E+01        2.245E-05        103.558
15           -2.9687718262E+01        1.124E-05        102.519
16           -2.9687717942E+01        7.377E-06        102.367
Total number of SCF: 16    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9687717942E+01 (Ha/atom)
Total free energy                  : -1.7812630765E+02 (Ha)
Band structure energy              : -4.2550145037E+01 (Ha)
Exchange correlation energy        : -2.4734406785E+01 (Ha)
Self and correction energy         : -3.5605006960E+02 (Ha)
Entropy*kb*T                       : -2.0225475673E-07 (Ha)
Fermi level                        : -1.6557659783E-01 (Ha)
Average force                      :  3.6393879962E-02 (Ha/Bohr)
Maximum force                      :  8.0418103132E-02 (Ha/Bohr)
Time for force calculation         :  30.337 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  2067.384 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
