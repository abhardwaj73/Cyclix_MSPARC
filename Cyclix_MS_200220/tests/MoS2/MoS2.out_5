***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 06-May-2020  Start time: 15:55:18                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.910213 0.369599 6.055851 
FD_GRID: 150 53 31
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 17 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 50
NSTATES: 36
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/MoS2/MoS2.out_5
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.199401 (Bohr)
Mesh spacing in y-direction        :  0.195493 (Bohr)
Mesh spacing in z direction        :  0.195350 (Bohr)
Number of symmetry adapted k-points:  9
Output printed to                  :  ../tests/MoS2/MoS2.out_5
Total number of atom types         :  2
Total number of atoms              :  6
Total number of electrons          :  52
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ./Mo.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.50 0.21 6.64
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ./S.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.50 0.21 6.64
Number of atoms of type 2          :  4
Estimated total memory usage       :  5.47 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
