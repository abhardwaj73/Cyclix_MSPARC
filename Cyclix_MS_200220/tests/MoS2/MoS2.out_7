***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 06-May-2020  Start time: 17:51:41                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.910213 0.369599 6.055851 
FD_GRID: 150 53 31
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 17 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 1
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 50
NSTATES: 36
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/MoS2/MoS2.out_7
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.199401 (Bohr)
Mesh spacing in y-direction        :  0.195493 (Bohr)
Mesh spacing in z direction        :  0.195350 (Bohr)
Number of symmetry adapted k-points:  9
Output printed to                  :  ../tests/MoS2/MoS2.out_7
Total number of atom types         :  2
Total number of atoms              :  6
Total number of electrons          :  52
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ./Mo.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.50 0.21 6.64
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ./S.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.50 0.21 6.64
Number of atoms of type 2          :  4
Estimated total memory usage       :  10.26 GB
========================================================================================
                            Self Consistent Field (SCF#1)                     
========================================================================================
Iteration     Free Energy (Ha/atom)    Magnetization     SCF Error        Timing (sec)
1            -2.9561301931E+01         4.0036E+00        1.787E-01        886.705
2            -2.9581740075E+01         3.3189E+00        3.209E-01        230.662
3            -2.9646251095E+01         2.9669E+00        6.139E-02        230.531
4            -2.9660437409E+01         2.0327E+00        3.341E-02        225.716
5            -2.9670342290E+01         1.1950E+00        2.295E-02        224.570
6            -2.9682639801E+01         1.8262E-05        2.614E-02        224.040
7            -2.9685149498E+01         1.8262E-05        1.439E-02        220.869
8            -2.9686481944E+01         1.8262E-05        7.482E-03        223.400
9            -2.9687192068E+01         1.8240E-05        3.393E-03        220.487
10           -2.9687513559E+01         1.5632E-05        3.722E-03        220.234
11           -2.9687677560E+01         3.9445E-06        1.672E-03        217.466
12           -2.9687700938E+01         3.1826E-06        1.171E-03        215.689
13           -2.9687713188E+01        -1.7296E-06        5.034E-04        215.798
14           -2.9687716200E+01         6.1320E-07        2.776E-04        214.690
15           -2.9687717387E+01         2.0617E-07        1.742E-04        226.322
16           -2.9687718927E+01         8.6149E-09        6.690E-05        218.455
17           -2.9687718328E+01        -1.2976E-08        3.446E-05        212.972
18           -2.9687718202E+01        -1.2795E-07        2.656E-05        213.647
19           -2.9687717679E+01        -1.7690E-07        1.887E-05        221.642
20           -2.9687717582E+01        -1.8297E-07        1.247E-05        212.968
21           -2.9687717697E+01        -1.5593E-07        7.991E-06        212.816
Total number of SCF: 21    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9687717697E+01 (Ha/atom)
Total free energy                  : -1.7812630618E+02 (Ha)
Band structure energy              : -4.2550066636E+01 (Ha)
Exchange correlation energy        : -2.4734400422E+01 (Ha)
Self and correction energy         : -3.5605006960E+02 (Ha)
Entropy*kb*T                       : -2.0225493077E-07 (Ha)
Fermi level                        : -1.6557627958E-01 (Ha)
Net Magnetization                  : -1.5592894442E-07 
Average force                      :  3.6385440004E-02 (Ha/Bohr)
Maximum force                      :  8.0423608610E-02 (Ha/Bohr)
Time for force calculation         :  58.857 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  5390.469 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
