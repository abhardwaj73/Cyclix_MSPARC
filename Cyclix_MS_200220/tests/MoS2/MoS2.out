***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 05-May-2020  Start time: 11:28:51                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.910213 0.369599 6.055851 
FD_GRID: 150 53 31
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 17 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 50
NSTATES: 36
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/MoS2/MoS2.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.199401 (Bohr)
Mesh spacing in y-direction        :  0.195493 (Bohr)
Mesh spacing in z direction        :  0.195350 (Bohr)
Number of symmetry adapted k-points:  9
Output printed to                  :  ../tests/MoS2/MoS2.out
Total number of atom types         :  2
Total number of atoms              :  6
Total number of electrons          :  52
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ./Mo.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.50 0.21 6.64
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ./S.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.50 0.21 6.64
Number of atoms of type 2          :  4
Estimated total memory usage       :  5.47 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9644940424E+01        1.017E-01        410.913
2            -2.9673280273E+01        1.135E-01        110.733
3            -2.9686225461E+01        2.278E-02        107.827
4            -2.9686991015E+01        2.099E-02        108.591
5            -2.9687354085E+01        1.160E-02        104.488
6            -2.9687479100E+01        3.932E-03        111.026
7            -2.9687476697E+01        2.821E-03        107.311
8            -2.9687494856E+01        1.589E-03        102.656
9            -2.9687496629E+01        6.170E-04        105.353
10           -2.9687495186E+01        2.389E-04        103.756
11           -2.9687494685E+01        1.884E-04        100.143
12           -2.9687494254E+01        1.036E-04        103.748
13           -2.9687493984E+01        3.521E-05        101.622
14           -2.9687492914E+01        2.380E-05        100.898
15           -2.9687493915E+01        1.498E-05        99.299
16           -2.9687493137E+01        6.973E-06        98.652
Total number of SCF: 16    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9687493137E+01 (Ha/atom)
Total free energy                  : -1.7812495882E+02 (Ha)
Band structure energy              : -4.2569144488E+01 (Ha)
Exchange correlation energy        : -2.4733241875E+01 (Ha)
Self and correction energy         : -3.5605006960E+02 (Ha)
Entropy*kb*T                       : -2.0234917134E-07 (Ha)
Fermi level                        : -1.6591672038E-01 (Ha)
Average force                      :  3.8919998560E-02 (Ha/Bohr)
Maximum force                      :  8.3236358799E-02 (Ha/Bohr)
Time for force calculation         :  30.828 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  2047.861 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
